StyledStringChains/NSAttributedString+stylable.swift

```diff
@@ -94,6 +94,14 @@ extension NSMutableAttributedString: Stylable {
         return addAttribute(.writingDirection, value: value)
     }
 
+    public func withSpace(_ count: Int) -> Stylable {
+        let x: String =  (0..<count).map { _ in " " }.joined()
+        return (self ++ x)
+    }
+
+    public func withDelimiter(_ delimiter: String) -> Stylable {
```

Comment:
It doesn't look right to me. 

In general, when you set delimiter and provide a collection, it will merge the collection with delimiter between each item. Here, it is just adding only once. 

So, can we be more specific? like `addText:`, `tab(count: )` and `newLines(count: )`?

Reply:
Yeah. I agree `delimiter` is used for some other purpose. I'll change it to something else.
Adding more functions for a tab, newLines doesn't look scalable to me as it can keep on growing. I'll have only one function which accepts a string.

Reply:
True. These methods which I suggested are like syntactic sugars. So developer don't have to pass in "\n", "\t". If we provide addText, then still they can pass those strings. These methods are just handy to use.